source/process/support.md

```diff
@@ -180,13 +180,23 @@ echo | openssl s_client -showcerts -connect <URL>:443 -CApath /etc/ssl/ && echo
 
 #### Other debugging information
 
-Useful commands (may need to be run with sudo):
-- `gitlab-ctl reconfigure` - Update config.json and other configuration files, then restart all services (including Mattermost).
-- `gitlab-ctl <stop/start/restart> mattermost` - Stop/start/restart Mattermost.
-- `gitlab-ctl status mattermost` - Check if Mattermost is running. The message printed will start with `run` if it's running or `down` if it's not.
-- `gitlab-ctl tail mattermost` - Watch all Mattermost log files. Press Ctrl+C to exit.
-- `gitlab-ctl <stop/start/restart/status/tail> <nginx/postgresql/redis/etc>` - Control, view status, or view logs of a different service.
-- `sudo gitlab-psql -d mattermost_production` - Access the embedded Mattermost database
+Useful commands:
+- `sudo /opt/gitlab/bin/gitlab-ctl reconfigure` - Update `config.json` and other configuration files, then restart all services (including Mattermost).
```

Comment:
The full path to each of these shouldn't be necessary. `gitlab-ctl` will be on the PATH, so these can just be run as
```suggestion
- `sudo gitlab-ctl reconfigure` - Update `config.json` and other configuration files, then restart all services (including Mattermost).
```